.github/scripts/toolchain/build-gcc.sh

```diff
@@ -12,52 +12,72 @@ if [ "$RUN_CONFIG" = 1 ] || [ ! -f "$GCC_BUILD_PATH/Makefile" ] ; then
 
     rm -rf $GCC_BUILD_PATH/*
 
-    case $TARGET in
+    case $PLATFORM in
         *mingw*)
-            HOST_OPTIONS=" \
+            TARGET_CONF=" \
                 --enable-threads=win32 \
                 --disable-win32-registry"
             ;;
+        *linux*)
+            TARGET_CONF=" \
+                --enable-threads=posix"
+            ;;
+    esac
+
+    case $ARCH in
+        x86_64)
+            TARGET_CONF="$TARGET_CONF \
+                --with-arch=nocona \
+                --with-tune=generic"
+            ;;
+        aarch64)
+            TARGET_CONF="$TARGET_CONF \
+                --with-arch=armv8-a \
```

Comment:
This was proposed as a solution for #105. It does not work alone though. I'd say that it's a good idea to bind the binaries to particular instruction set to prevent the uncertainity/sudden changes anyway.